alf/environments/suite_babyai.py

```diff
@@ -172,8 +171,10 @@ def __init__(self,
             'mission':
                 gym.spaces.MultiDiscrete([vocab_size] * max_instruction_length)
         }
-        if one_token_per_step:
+        if mode == 'word':
             obs_space['mission'] = gym.spaces.Discrete(vocab_size)
+        elif mode == 'char':
+            obs_space['mission'] = gym.spaces.Discrete(128)
 
```

Comment:
Might be able to add a comment on using UTF-8 here or in the ```Arg:``` section?

Reply:
Added